app/models/agents/liquid_output_agent.rb

```diff
@@ -0,0 +1,214 @@
+module Agents
+  class LiquidOutputAgent < Agent
+    include WebRequestConcern
```

Comment:
I suspect this may be removed, as this agent does not seem to make any web request.